susemanager-utils/testing/automation/backend-unittest-oracle.sh

```diff
@@ -14,10 +14,10 @@ EXIT=0
 
 INITIAL_CMD="/manager/susemanager-utils/testing/automation/initial-objects.sh"
 CMD="/manager/backend/test/docker-backend-oracle-tests.sh"
-CLEAN_CMD="/manager/susemanager-utils/testing/automation/clean-objects.sh"
+CHOWN_CMD="/manager/susemanager-utils/testing/automation/chown-objects.sh $(id -u) $(id -g)""
```

Comment:
Incorrect ^, as there are two quotes :-)

Consecuence of using `sed`. I didn't see at the script I used for the test as that was the one I changed manually.

Fixing it right now.